src/offset_date_time.rs

```diff
@@ -546,6 +546,110 @@ impl OffsetDateTime {
     }
 }
 
+/// Methods that replace the following components `Date`, `Time`, `PrimitiveDateTime`, `Offset`.
+impl OffsetDateTime {
+    /// Replacing with a new `Time` component. The new `PrimitiveDateTime` value is stored as UTC internally.
+    ///
+    /// The return value is a `Offset_date_time`.
+    ///
+    /// ```rust
+    ///   # use time_macros::{datetime,offset,time};
+    /// assert_eq!(
+    ///     datetime!("2020-01-01 5:00")
+    ///         .assume_offset(offset!("UTC"))
```

Comment:
The `datetime!` macro accepts a `UtcOffset` after the time. This is admittedly quite new; I don't believe it accepted anything other than UTC when you created the PR.

Reply:
Should 'cargo test' detect this one? What should I do with this one?